src/index.ts

```diff
@@ -17,11 +17,12 @@
 import * as common from '@google-cloud/common-grpc';
 import {paginator} from '@google-cloud/paginator';
 import {replaceProjectIdToken} from '@google-cloud/projectify';
-import {promisifyAll} from '@google-cloud/promisify';
+import {callbackifyAll, promisifyAll} from '@google-cloud/promisify';
```

Comment:
are you still using `promisifyAll` here?

Reply:
No, not anymore. Will remove it, thanks.